common/ux-window.cpp

```diff
@@ -113,7 +117,7 @@ namespace rs2
         // On Mac-OS, mixing OpenGL 2 with OpenGL 3 is not supported by the driver
         // while this can be worked-around, this will take more development time,
         // so for now Macs should not use the GLSL stuff
-        config_file::instance().set_default(configurations::performance::glsl_for_processing, false);
+        //config_file::instance().set_default(configurations::performance::glsl_for_processing, false);
```

Comment:
I comment out the placed that set it because it has special logic if it's MAC or not

Reply:
I am not sure about removing it or keep it commented out, thoughts?

Reply:
Don't remove. It's a historical investigation we won't have time and resources to re-investigate if users will complain about again. I would even leave uncommented.